server/backend/api/app/crud/crud_user_role.py

```diff
@@ -143,7 +144,7 @@ def create_user_role_assignment_many(
             )
             # Create user/role assignment
             new_user_role_assginment_res = create_user_role_assignment(
-                db, fam_user, child_role, requester.cognito_user_id,
+                db, fam_user, child_role, requester.cognito_user_id, expiry_date=request._expiry_date
```

Comment:
Where do we validate what was supplied for expiry date via the API argument?

Reply:
Hi Basil, the validation is on a Pydantic Schema class: server/backend/api/app/schemas/fam_user_role_assignment_create.py. This is used for endpoint body argument intake.